beacon-chain/powchain/service.go

```diff
@@ -88,6 +88,10 @@ type ChainStartFetcher interface {
 type ChainInfoFetcher interface {
 	Eth2GenesisPowchainInfo() (uint64, *big.Int)
 	IsConnectedToETH1() bool
+	CurrentEndpoint() string
```

Comment:
Recommend calling these CurrentETH1Endpoint for extra clarity, same with Endpoints() becoming ETH1Endpoints()